internal/controllers/rewards.go

```diff
@@ -198,17 +198,19 @@ func (r *RewardsController) GetUserRewards(c *fiber.Ctx) error {
 
 		pts := 0
 		for _, r := range rewards {
-			pts += r.StreakPoints + r.IntegrationPoints
+			pts += r.StreakPoints + r.AftermarketDevicePoints + r.SyntheticDevicePoints
 		}
 
 		userPts += pts
 
 		tkns := big.NewInt(0)
 		for _, t := range rewards {
-			if t.Tokens.IsZero() {
+			if t.AftermarketDeviceTokens.IsZero() && t.SyntheticDeviceTokens.IsZero() && t.StreakTokens.IsZero() {
```

Comment:
Okay, looking [here](https://github.com/DIMO-Network/rewards-api/blob/7ba93e6d587a7e9b127474852af24a002be73356/internal/storage/db.go#L22C34-L22C34) it seems like these are either all set or all not set. Maybe leave a comment about that? On a first read it's a bit alarming.